docs/src/examples/components/Dropdown/Types/DropdownExampleMultipleSearch.shorthand.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react'
+import { Dropdown, DropdownListItem } from '@stardust-ui/react'
+
+class DropdownExample extends React.Component {
+  state = {
+    items: inputItems,
+  }
+  render() {
+    return (
+      <Dropdown
+        multiple
+        search
+        placeholder="Start typing a name"
+        items={this.state.items}
+        onDropdownChange={(value: DropdownListItem[]) => {
+          this.setState({
+            items: inputItems.filter(item => value.indexOf(item) === -1),
+          })
+        }}
+      />
+    )
+  }
+}
+
+const inputItems: DropdownListItem[] = [
```

Comment:
I know, it's just that so far we have 3,4 array items, and I chose to have 8-9. I thought it will be weird for the user to pop-up the editor and scroll to the very bottom to see the actual usage.

If you think it's still better than the current approach I will address it.

Reply:
no, you're right, you can leave as is but don't revert if you already made the change